sunpy/visualization/drawing.py

```diff
@@ -194,6 +194,54 @@ def prime_meridian(axes, *, rsun: u.m = R_sun, resolution=500, **kwargs):
     return visible, hidden
 
 
+@u.quantity_input
+def extent(axes, wcs, *, rsun: u.m = R_sun, **kwargs):
```

Comment:
Without having looked at this in detail, this feels like something which could be useful to upstream to astropy?